dashboard-ui/src/pages/home/log-metadata-provider.tsx

```diff
@@ -155,9 +161,9 @@ export const LogMetadataProvider = ({ kubeContext }: LogMetadataProviderProps) =
     return subscribeToMore({
       document: LOG_METADATA_LIST_WATCH,
       updateQuery: (prev, { subscriptionData }) => {
-        const ev = subscriptionData.data.logMetadataWatch;
+        const ev = (subscriptionData.data as any)?.logMetadataWatch;
```

Comment:
You shouldn't need to cast as `any` or use an optional chaning operator here.
```suggestion
        const ev = subscriptionData.data.logMetadataWatch;
```